xroad-catalog-lister/src/main/java/org/niis/xroad/catalog/lister/service/CatalogServiceImpl.java

```diff
@@ -355,11 +356,16 @@ public Boolean checkDatabaseConnection() {
     @Override
     public LastCollectionData getLastCollectionData() {
         return LastCollectionData.builder()
-                .membersLastFetched(memberRepository.findLatestFetched())
-                .openapisLastFetched(openApiRepository.findLatestFetched())
-                .servicesLastFetched(serviceRepository.findLatestFetched())
-                .subsystemsLastFetched(subsystemRepository.findLatestFetched())
-                .wsdlsLastFetched(wsdlRepository.findLatestFetched()).build();
+                .membersLastFetched(memberRepository.findLatestFetched() == null ? null
+                        : LocalDateTime.ofInstant(serviceRepository.findLatestFetched(), ZoneId.systemDefault()))
+                .openapisLastFetched(openApiRepository.findLatestFetched() == null ? null
+                        : LocalDateTime.ofInstant(openApiRepository.findLatestFetched(), ZoneId.systemDefault()))
+                .servicesLastFetched(serviceRepository.findLatestFetched() == null ? null
+                        : LocalDateTime.ofInstant(serviceRepository.findLatestFetched(), ZoneId.systemDefault()))
+                .subsystemsLastFetched(subsystemRepository.findLatestFetched() == null ? null
+                        : LocalDateTime.ofInstant(serviceRepository.findLatestFetched(), ZoneId.systemDefault()))
+                .wsdlsLastFetched(wsdlRepository.findLatestFetched() == null ? null
+                        : LocalDateTime.ofInstant(wsdlRepository.findLatestFetched(), ZoneId.systemDefault())).build();
```

Comment:
We should consider, if the base datatype in the DTO can be changed to `Instant` instead.

Alternatively, if it would be better to keep the DTO with `LocalDateTime`, I think this would be a good case for a utility function to improve readability and avoid the inline null checks.